packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -262,7 +258,7 @@ class TextSelectionOverlay {
     required this.renderObject,
     this.selectionControls,
     bool handlesVisible = false,
-    this.selectionDelegate,
+    required this.selectionDelegate,
```

Comment:
This is probably an error when converting to null safety. It will likely crash the app if this is null

Reply:
I agree, looks like it should have been `required` originally.